legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/auth/GitLabOAuthAuthenticator.java

```diff
@@ -346,16 +370,16 @@ private String getAccessTokenFromLocation(String location, String expectedState)
             throw new GitLabAuthOtherException("Could not get access token from redirect URI " + location + ": could not parse parameters from fragment", e);
         }
 
-        String accessToken = null;
+        String code = null;
         boolean foundState = false;
         for (NameValuePair nvp : parameters)
         {
             switch (nvp.getName())
             {
-                case ACCESS_TOKEN_PARAM:
+                case CODE_PARAM:
                 {
-                    accessToken = nvp.getValue();
-                    break;
+                        code = nvp.getValue();
+                        break;
```

Comment:
This got over-indented.
```suggestion
                    code = nvp.getValue();
                    break;
```